test/integration/results_test.exs

```diff
@@ -41,3 +41,40 @@ defmodule ResultsTest do
     assert inspect(page) == ~s(#Xandra.Page<[rows: [%{"name" => "Jeff"}], more_pages?: false]>)
   end
 end
+
+defmodule ResultsAtomTest do
+  use XandraTest.IntegrationCase, async: true, start_options: [atom_keys: true]
+
+  alias Xandra.{SchemaChange, SetKeyspace, Void}
+
+  test "each possible result", %{conn: conn, keyspace: keyspace} do
+    assert {:ok, result} = Xandra.execute(conn, "USE #{keyspace}", [])
+    assert result == %SetKeyspace{keyspace: String.downcase(keyspace)}
+
+    statement = "CREATE TABLE numbers_atoms (figure int PRIMARY KEY)"
+    assert {:ok, result} = Xandra.execute(conn, statement, [])
+    assert result == %SchemaChange{
+      effect: "CREATED",
+      options: %{
+        keyspace: keyspace |> String.downcase() |> String.to_atom(),
```

Comment:
I am not sure returning all metadata as keys is a good idea. My initial understanding was that we would only return column names as atoms. Making all metadata atoms works fine but it might be confusing to document and users might be surprised by this behaviour. Wdyt @lexmag?

Reply:
I thought `atom_keys` will only affect "rows" response.
It seems more intuitive to me—not sure how practical that is to return atom values for keyspace.